### PR TITLE
ci: add code owners for chaos packages

### PR DESCRIPTION
**Pull request by Augustyniak**

### CODE REVIEW

**Comment by Augustyniak** [comment]
@wgallagher Do you think that there are any more directories that our team should own?

**Comment by danielhochman** [comment]
api definitions?

**Comment by Augustyniak** [comment]
> api definitions?

api definitions are in `/backend/api/chaos` which should be covered by `/backend/**/chaos` from the CODEOWNERS file

**Comment by danielhochman** [comment]
they're in https://github.com/lyft/clutch/tree/main/api/chaos

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1,4 @@
-*       @lyft/clutch-admin
+*                                            @lyft/clutch-admin
+/backend/**/chaos/                           @lyft/resilience
```
Comment (dschaller) [inline .github/CODEOWNERS:2]: Should this include `/api/chaos/` as well?

**Review by danielhochman — APPROVED** [review]

**Pull request merged**